lib/plugins/entities.js

```diff
@@ -135,25 +136,26 @@ function inject (bot, { version }) {
     // spawn object/vehicle
     const entity = fetchEntity(packet.entityId)
     let entityData = objects[packet.type]
-    if (entityData === undefined) {
+    if (entityData === undefined && bot.majorVersion !== '1.13') {
```

Comment:
This is wrong, don't define behavior by inclusion, you don't know it will work for any other version than the ones you tested